extensions/import/src/wizard/pages/deployPlanPage.ts

```diff
@@ -0,0 +1,296 @@
+			},
+			{
+				value: localize('dacfx.typeColumn', 'Type'),
+				width: 100,
+				cssClass: 'align-with-header',
+				toolTip: localize('dacfx.typeTooltip', 'Type of object that will be affected by deployment')
+			},
+			{
+				value: localize('dacfx.objectColumn', 'Object'),
+				width: 300,
+				cssClass: 'align-with-header',
+				toolTip: localize('dacfx.objecTooltip', 'Name of object that will be affected by deployment')
+			}];
+
+		if (dataloss) {
+			columns.unshift(
+				{
+					value: localize('dacfx.dataLossColumn', 'Data Loss'),
+					width: 50,
+					cssClass: 'center-align',
+					toolTip: localize('dacfx.dataLossTooltip', 'Operations that may result in data loss are marked with a warning sign')
+				});
+		}
+		return columns;
+	}
+
+	private parseXml(report: string): DeployPlanResult {
+		let operations = new Array<TableObject>();
+		let dataLossAlerts = new Set<string>();
+
+		let currentOperation = '';
+		let dataIssueAlert = false;
+		let currentReportSection: deployPlanXml;
+		let currentTableObj: TableObject;
+		let p = new parser.Parser({
+			onopentagname(name) {
+				if (name === deployPlanXml.AlertElement) {
+					currentReportSection = deployPlanXml.AlertElement;
+				} else if (name === deployPlanXml.OperationElement) {
+					currentReportSection = deployPlanXml.OperationElement;
+				} else if (name === deployPlanXml.ItemElement) {
+					currentTableObj = new TableObject();
+				}
+			},
+			onattribute: function (name, value) {
+				if (currentReportSection === deployPlanXml.AlertElement) {
+					switch (name) {
+						case deployPlanXml.NameAttribute: {
+							// only care about showing data loss alerts
+							if (value === deployPlanXml.DataIssueAttribute) {
+								dataIssueAlert = true;
+							}
+							break;
+						}
+						case deployPlanXml.IdAttribute: {
+							if (dataIssueAlert) {
+								dataLossAlerts.add(value);
+							}
+							break;
+						}
+					}
+				} else if (currentReportSection === deployPlanXml.OperationElement) {
+					switch (name) {
+						case deployPlanXml.NameAttribute: {
+							currentOperation = value;
+							break;
+						}
+						case deployPlanXml.ValueAttribute: {
+							currentTableObj.object = value;
+							break;
+						}
+						case deployPlanXml.TypeAttribute: {
+							currentTableObj.type = value;
+							break;
+						}
+						case deployPlanXml.IdAttribute: {
+							if (dataLossAlerts.has(value)) {
+								currentTableObj.dataloss = true;
+							}
+							break;
+						}
+					}
+				}
+			},
+			onclosetag: function (name) {
+				if (name === deployPlanXml.ItemElement) {
+					currentTableObj.operation = currentOperation;
+					operations.push(currentTableObj);
+				}
+			}
+		}, { xmlMode: true, decodeEntities: true });
+		p.parseChunk(report);
+
+		let data = new Array<Array<string>>();
+		operations.forEach(operation => {
+			let isDataLoss = operation.dataloss ? '⚠️' : '';
```

Comment:
OMG, what is this?

Reply:
confirmed with Alan that this is ok since it's part of unicode and the color looks ok with both dark and light theme